satellite/audit/worker.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+// Worker contains information for populating audit queue and processing audits.
+type Worker struct {
+	log    *zap.Logger
+	config Config
+	closed chan struct{}
+
+	queue *queue
+}
+
+// NewWorker instantiates Worker and workers.
```

Comment:
What's the difference between `Worker` and `workers` in the comment?

Reply:
Fixed